oio/cli/object/object.py

```diff
@@ -785,3 +785,57 @@ def take_action(self, parsed_args):
             account, parsed_args.container, parsed_args.object,
             maxvers=parsed_args.max_versions
         )
+
+class LinkObject(ObjectCommandMixin, command.Command):
+    """Make a shallow copy of an object."""
+
+    log = getLogger(__name__ + '.LinkObject')
+
+    def get_parser(self, prog_name):
+        parser = super(LinkObject, self).get_parser(prog_name)
+        self.patch_parser(parser)
+        parser.add_argument(
+            'linked_account',
+            metavar='<linked account>',
+            help='Name of the linked account.')
+        parser.add_argument(
+            'linked_container',
+            metavar='<linked container>',
+            help='Name of the linked container.')
+        parser.add_argument(
+            'linked_object',
+            metavar='<linked object>',
+            help='Name of the linked object.')
+        parser.add_argument(
+            '--content-id',
+            metavar='<content ID>',
+            help='Content ID.')
+        parser.add_argument(
+            '--linked-content-id',
+            metavar='<linked content ID>',
+            help='linked content ID.')
+        parser.add_argument(
+            '--properties-directive-replace',
+            help='Set properties directive to replace',
+            action='store_true',
+            dest= 'replace')
```

Comment:
You should use `--property`, like `openio object create`. If there is no property, you `COPY` the properties, otherwise you `REPLACE`.